### PR TITLE
Rename the command from 'gitlab-cli' to 'gitlab'

### LINKED ISSUE #25: Remove -cli from the executable name?
Hello, any reason to append `-cli` to the executable name? We know it’s a CLI tool, since we use it… from the CLI. Typing `$ gitlab projects` would seem more natural than `$ gitlab-cli projects`.

Thoughts?

### PR DESCRIPTION
**Pull request by bfontaine**
closes #25.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
 /gitlab-cli
 /gitlab-cli_v*
```
Comment (makkes) [inline .gitignore]: Those should then be removed, shouldn't they?

Comment (bfontaine) [reply .gitignore]: Yep, I removed them.

**Comment by bfontaine** [comment]
I changed the `.github/workflow/release.yaml`.

**Comment by makkes** [comment]
I'll run the unit tests manually and clean up the Github Actions mess afterwards. Sorry for that.

**Pull request merged**